data/mods/Magiclysm/monsters/Orcs.json

```diff
@@ -0,0 +1,103 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior.  It stares at you menacingly, sword in hand.",
+    "default_faction": "orcs",
+    "bodytype": "human",
+    "species": [ "ORC" ],
+    "volume": "60 L",
+    "weight": "70 kg",
+    "hp": 92,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "o",
+    "color": "green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 6,
+    "melee_dice_sides": 4,
+    "dodge": 7,
+    "harvest": "orc",
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK" ],
+    "vision_day": 13,
+    "vision_night": 5,
+    "zombify_into": "mon_zorc",
+    "death_drops": {
+      "subtype": "collection",
+      "items": [
+        { "item": "helmet_scrap", "prob": 20 },
+        { "item": "legguard_scrap", "prob": 20 },
+        { "item": "boots_scrap", "prob": 20 },
+        { "item": "armguard_scrap", "prob": 20 },
+        { "item": "cuirass_scrap", "prob": 10 },
+        { "item": "longsword", "prob": 100 }
```

Comment:
I just noticed that they don't have melee_cut, unlike the blood orc. Perhaps swap which one gets a longsword and which one gets the defense_caravan_melee?

Reply:
Note that NC_THUG_bashing is an alternative to defense_caravan_melee (I didn't spot it earlier because it's in the npcs directory, not itemgroups).

Reply:
I'll add melee_cut to orc warrior.

Reply:
Melee_cut has been added to orc warrior.